mthamayor/auto-mart
# Finishes #166285419 Added authorization tokens to requests

#### What does this PR do?
Added JWT Authorization token to all requests

#### Description of Task to be completed?
The tests used dummy tokens when requests were being sent.
This means user data was sent through the form body, and not the token payload
A revamp was made to make sure protected endpoints use **valid** tokens and the data, formally gotten from the form body was now removed and the JWT payload is now being used instead
Unit tests were refactored to adhere to this new development.

#### How should this be manually tested?
After cloning the repo

Start the development server with
npm run dev
After signing in, and you want to access a protected route,
Copy your Authorization token, 
On postman, set the Authorization header of the route you want to access with the value of your token
Bearer <token>
The form body should contain the necessary fields

#### Any background context you want to provide?
Developments and test environments now require a valid authorization token to access protected endpoints

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166285419
